doc/releases/index.rst

```diff
@@ -44,6 +44,20 @@ following releases:
 For more information, see  :ref:`Security Vulnerability Reporting <reporting>`.
 
 
+Supported Releases
+******************
+
++------------+----------------+---------------+
+| Release    | Release date   | EOL           |
++============+================+===============+
+| 2.7.4      | 23/12/2022     | ~ 08/2024     |
++------------+----------------+---------------+
+| 3.3.0      | 19/02/2023     | ~ 10/2023     |
++------------+----------------+---------------+
+| 3.2.0      | 30/09/2022     | ~ 06/2023     |
```

Comment:
Ah, I see you put this table here, great!

My only request is that we stop being wishy-washy with `~` and set defined dates, which may be extended, but which will never be pulled in.